WindowsServerDocs/networking/sdn/technologies/network-controller/Network-Controller.md

```diff
@@ -22,7 +22,7 @@ Using Network Controller, you can automate the configuration of network infrastr
 > - [Deploy Network Controller using Windows PowerShell](../../deploy/Deploy-Network-Controller-using-Windows-PowerShell.md)
 > - [Install the Network Controller server role using Server Manager](Install-the-Network-Controller-server-role-using-Server-Manager.md)
 > - [Post-Deployment Steps for Network Controller](post-deploy-steps-nc.md)
-> - [Network Controller Cmdlets](https://technet.microsoft.com/library/mt576401.aspx)
+> - [Network Controller Cmdlets](https://docs.microsoft.com/en-us/powershell/module/networkcontroller/?view=win10-ps)
```

Comment:
This link needs to have the locale removed so it works properly with localization. It should be: https://docs.microsoft.com/powershell/module/networkcontroller/?view=win10-ps

Reply:
You actually need to make the change to your branch and resubmit. I can't do it.